themes/custom/droopler_theme/templates/product/page--products.html.twig

```diff
@@ -68,6 +68,15 @@
 
 {% block content %}
 
+  <div class="container-fluid">
```

Comment:
We should rather use `container` here, check what happens if the title is extremely long - it should be the same width as page header (the one with logo and menu)